.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "julia.environmentPath": "c:\\Users\\Dan\\Documents\\Uni\\Year 4\\Project\\GitHub\\AccretionFormulae.jl"
+}
```

Comment:
`.vscode` shouldn't be committed unless for good reason. Add this to your personal `.gitignore` file to avoid accidentally committing it.

Reply:
I think I must have accidentally clicked on something, which made this turn up, I think I should have gotten rid of it now.